addons/website/models/website_page.py

```diff
@@ -212,7 +212,7 @@ def _get_cache_key(self, req):
         # Always call me with super() AT THE END to have cache_key_expr appended as last element
         # It is the only way for end user to not use cache via expr.
         # E.g  (None if 'token' in request.params else 1,)  will bypass cache_time
-        cache_key = (req.website.id, req.lang, req.httprequest.path)
+        cache_key = (req.website.id, req.lang, req.session.geoip.get('country_code'), req.httprequest.path)
```

Comment:
what's your opinion about move this in session info @qsm-odoo  ? and avoid to fork cache ? 
This one has is removed in chm PR so it will be probably quickly broekn with their new cache...